controllers/UserController.js

```diff
@@ -486,13 +491,36 @@ export default class UserController {
         res.status(201).json(
           new ResponseBuilder()
             .setData(response)
+            .setMessage('We are reviewing your process. Thank you.')
```

Comment:
Ini lebih ke bahasanya sih `We'll be reviewing your proposal and respond very soon. Thank you` harusnya gitu aja.

Reply:
oke. kuganti ini ya